cmd/degitx/config.go

```diff
@@ -25,7 +25,7 @@ type NodeConfig struct {
 	LogConfig *logging.LogConfig `yaml:"logging"`
 }
 
-func (config *NodeConfig) fromFile(fileName string) error {
+func (config *NodeConfig) FromFile(fileName string) error {
```

Comment:
@stepanvalyavskiy if we reuse configuration parsing code from two `cmd` applications, it's better to move it to upper level from `cmd/`. I'd suggest using `internal/` directory, since configuration parsing doesn't seems to be the part of public API: https://github.com/golang-standards/project-layout/tree/master/internal